tools/deb/istio-iptables.sh

```diff
@@ -30,19 +28,26 @@
 # After more testing, the goal is to replace and unify the script in K8S - by generating
 # the sidecar image using the .deb file created by proxy.
 
+set -o errexit
```

Comment:
in short, this makes it fail because it's doing -D comments that are expected to fail; those commands should have || true